src/api/FileCollaborators.js

```diff
@@ -8,6 +8,7 @@ import MarkerBasedAPI from './MarkerBasedAPI';
 import { DEFAULT_MAX_COLLABORATORS } from '../constants';
 import type { ElementsErrorCallback } from '../common/types/api';
 import type { SelectorItem, SelectorItems, UserMini, GroupMini } from '../common/types/core';
+import type { Collaborators } from '../elements/content-sidebar/flowTypes';
```

Comment:
We want to try to avoid importing component modules into our API layer. Can we pull up this type to the "common" modules?